GRC Updates - Draft 2 (#3870)
open-cluster-management/rhacm-docs: governance/etcd_encryption_policy.adoc

                metadata:
                  name:
                spec:
                  encryption:
                ...
----

[#etcd-encryption-policy-table]
== ETCD encryption policy table

|===
| Field | Optional or required | Description

| `apiVersion`
| Required
| Set the value to `policy.open-cluster-management.io/v1`.

| `kind`
| Required
| Set the value to `Policy` to indicate the type of policy.

| `metadata.name`
| Required
| The name for identifying the policy resource.

| `metadata.namespace`
| Required
| The namespace of the policy.

| `spec.remediationAction`
| Optional
| Specifies the remediation of your policy. The parameter values are `enforce` and `inform`. This value is optional since it will override any values provided in `spec.policy-templates`.

| `spec.disabled`
| Required
| Set the value to `true` or `false`.
The `disabled` parameter provides the ability to enable and disable your policies.

| `spec.policy-templates[].objectDefinition`
| Required
| Used to list configuration policies containing Kubernetes objects that must be evaluated or applied to the managed clusters.
|===

[#etcd-encryption-policy-sample]
== Etcd encryption policy sample

> See https://github.com/stolostron/policy-collection/blob/main/stable/SC-System-and-Communications-Protection/policy-etcdencryption.yaml[`policy-etcdencryption.yaml`] for the policy sample. See xref:../governance/create_policy.adoc#managing-security-policies[Managing security policies] for more information. Refer to the xref:../governance/policy_example.adoc[Policy YAML documentation], and to xref:../governance/config_policy_ctrl.adoc#kubernetes-configuration-policy-controller[Kubernetes configuration policy controller] to view other configuration policies that are monitored by the controller.

Author: I added the links to highlight that there were more details and fields (like `namespaceSelector` and `evaluationInterval`). How about this wording?
```suggestion
See https://github.com/stolostron/policy-collection/blob/main/stable/SC-System-and-Communications-Protection/policy-etcdencryption.yaml[`policy-etcdencryption.yaml`] for the policy sample. See the xref:../governance/policy_example.adoc[Policy YAML documentation] and the xref:../governance/config_policy_ctrl.adoc#kubernetes-configuration-policy-controller[Kubernetes configuration policy controller] to view additional details on policy and configuration policy fields.
```
